learning_journal/routes.py

```diff
@@ -5,7 +5,7 @@
 def includeme(config):
     """includes our views I guess..."""
     config.add_static_view('static', 'static', cache_max_age=3600)
-    config.add_route("home", "/")
-    config.add_route("edit", "/edit")
+    config.add_route("index", "/")
+    config.add_route("edit_existing", "/edit")
     config.add_route("form", "/form")
-    config.add_route("detail", "/detail/{id:\d+}")
+    config.add_route("entry_details", "/entry_details/{id:\d+}")
```

Comment:
see here we are using this *placeholder* in your URL to pass along the ID of the entry whose detail you wish to see.